include/CFL/CFLGraphBuilder.h

```diff
@@ -61,11 +79,11 @@ class CFLGraphBuilder
 
     /// Build Bidirectional graph by copying nodes and edges from any graph inherited from GenericGraph
     template<class N, class E>
-    CFLGraph* buildBigraph(GenericGraph<N,E>* graph)
+    CFLGraph* buildBigraph(GenericGraph<N,E>* graph, Kind startKind)
     {
-        CFLGraph *cflGraph = new CFLGraph();
+        CFLGraph *cflGraph = new CFLGraph(startKind);
```

Comment:
Make sure every time you create new memory via new, the memory will be released at some point like in descructors.

Reply:
Yes, the cflGraph memory object being pass to cflalias, and will be release in cflalias destructor.

Reply:
The normalised grammar and cflgraph should be released in the destructor of cflsolver.